src/pages/home.js

```diff
@@ -1,23 +1,43 @@
 import NavBar from "../components/navBar";
+import Footer from "../components/footer";
+import Question from "../assets/Question_of_the_week.png";
+import Friend from "../assets/Question_of_the_week.png";
 import { Link } from "react-router-dom";
 
 function Home() {
   return (
-    <div className="bg-cod-gray absolute h-full w-full font-audioWide">
+    <div className="bg-cod-gray absolute h-full w-full">
```

Comment:
`absolute` positioning is not a good way of creating a normal layout, it's suitable for things that are outside the normal flow of the page.